### PR TITLE
Closes #024 - admin-view-articles-list

### LINKED ISSUE #24: Login layout

### PR DESCRIPTION
**Pull request by dmytro-balabukh**

### CODE REVIEW

**Review by Rev1kon — APPROVED** [review]
amazing

**Review by ignars3 — APPROVED** [review]

**Review by BorysKopytko — APPROVED** [review]
Everything is OK.

File: Lanthanum.Web/Controllers/AdminController.cs [hunk 1]
```diff
@@ -1,16 +1,85 @@
...
+
+        public AdminController(IAdminService adminService)
+        {
+            _adminService = adminService;
+        }
+
         [HttpGet]
-        public IActionResult Article() => View();
+        [Route("articles-list")]
+        public async Task<ActionResult<AdminArticleViewModel>> ArticlesList(string filterConference, string filterTeam, string filterStatus, string searchString)
```
Comment (oleksandrmanetskyi) [inline Lanthanum.Web/Controllers/AdminController.cs]: If you create async method then name it ArticlesList**Async**

File: Lanthanum.Web/Controllers/AdminController.cs [hunk 2]
```diff
@@ -1,16 +1,85 @@
...
+            catch (Exception e)
+            {
+                //TODO Logging
+                throw;
+            }
+        }
+
+        [Route("delete-article"), ActionName("DeleteArticle")]
+        [HttpPost]
+        public async Task<ActionResult> DeleteArticleAsync(int id)
```
Comment (oleksandrmanetskyi) [inline Lanthanum.Web/Controllers/AdminController.cs:41]: `[Autorize(Role = "Admin")]` ?

**Review by bohdanburda — APPROVED** [review]

**Pull request merged**